frontend/packages/topology/src/components/page/TopologyEmptyState.tsx

```diff
@@ -21,22 +21,27 @@ const TopologyEmptyState: React.FC<TopologyEmptyStateProps> = ({ setIsQuickSearc
   return (
     <EmptyState className="odc-topology__empty-state" variant={EmptyStateVariant.full}>
       <EmptyStateIcon variant="container" component={TopologyIcon} />
-      <Title headingLevel="h3" size="lg">
+      <Title headingLevel="h3" size="lg" data-test="no-resources-found">
```

Comment:
If possible, please move this 2 lines up to the EmptyState as the complete component define this.

```tsx
    <EmptyState
      className="odc-topology__empty-state"
      variant={EmptyStateVariant.full}
      data-test="empty-state"
    >
      <EmptyStateIcon variant="container" component={TopologyIcon} />
      <Title headingLevel="h3" size="lg">
```

Reply:
Updated